structure.md

```diff
@@ -0,0 +1,29 @@
+# Structure
+
+Having a good structure is essential for our culture. We derive and maintain our structure from the learnings of the open-source community to maximize the odds to [better decisions](decision-making.md), innovation, and sustainability.
+
+Presently, our structure constitutes of the following supporting natures.
+
+## Flat
+
+The flat structure encourages members, regardless of who they are, to have an equal opportunity to contribute to the projects that suit their passions and interests. Such a structure plays a vital role in how we make decisions, collaborate, and communicate.
+
+Within the flat structure, members are naturally encouraged to contribute with minimal supervision. However, this also means that members are encouraged to provide support, guidance, and mentorship to other members. Specifically for new members, existing members would support with a more tailored guide and progressively transition them to be more independent and proactive to contribute creative solutions or even new problems that align with the business objectives and strategies.
```

Comment:
2nd sentence: 
However, this also means that more experienced members are encouraged to provide support, guidance, and mentorship to the newer ones.
3rd sentence: 
The more seasoned members will support the newer ones with a tailored and structured process that will transition them to independant work that aligns with the business objectives and strategies of the team.